app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command):
+    file = command.split()
+    if file[1] != file[2]:
+        with open(file[1], "r") as file1:
```

Comment:
`with open(..., "r") as file_in, open(..., "w") as file_out:`
Pls, use this construction